src/main/java/org/broadinstitute/hellbender/utils/recalibration/RecalibrationArgumentCollection.java

```diff
@@ -99,6 +99,9 @@
     @Argument(fullName = "preserve_qscores_less_than", shortName = "preserveQ", doc = "Don't recalibrate bases with quality scores less than this threshold (with -" + StandardArgumentDefinitions.BQSR_TABLE_SHORT_NAME + ")", optional = true)
     public int PRESERVE_QSCORES_LESS_THAN = QualityUtils.MIN_USABLE_Q_SCORE;
 
+    @HiddenOption
```

Comment:
Use `@Hidden` rather than `@HiddenOption` (neither actually hides the argument yet, but GATK3 used `@Hidden`, and sticking with that might make the GATKDocs code easier to port)